core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityService.scala

```diff
@@ -149,10 +149,9 @@ class EntityService(protected val userInfo: UserInfo, val dataSource: SlickDataS
     }
 
```

Comment:
Also out of scope perhaps? But EntityServiceSpec does not seem to have test coverage for a lot of the methods in this class